CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+# THIS LOG HAS BEEN DEPRECATED AS OF v2.6.4. PLEASE REFER TO OUR RELEASES INSTEAD.
```

Comment:
Would be good to have the URL to the releases page here for people who have cloned locally

```suggestion
# THIS LOG HAS BEEN DEPRECATED AS OF v2.6.4. [PLEASE REFER TO OUR RELEASES INSTEAD.](https://github.com/LearningLocker/learninglocker/releases)
```